data/assets/blog-stylesheet.scss

```diff
@@ -124,3 +124,14 @@ footer {
         }
     }
 }
+
+@media only print {
+    // Display the URL of external sources:
+    a[href^=http]:after {
+	content:" <" attr(href) ">";
+    }
+
+    ul, img, table {
+	page-break-inside: avoid;
+    }
+}
```

Comment:
+1, I tried a mixin to avoid duplication but without luck, it seems mixins don't work with media queries. But adding the media print in the base is the way to go. I don't know why these stylesheets have so many duplicated code already.